spring-cloud-dataflow-server-core/src/main/resources/META-INF/application-stream-common-properties-defaults.yml

```diff
@@ -0,0 +1,30 @@
+common:
+  management:
+    metrics:
+      tags:
+        # Common Spring Cloud Data Flow stream app tags.
+        stream.name: ${spring.cloud.dataflow.stream.name:unknown}
+        application.name: ${vcap.application.application_name:${spring.cloud.dataflow.stream.app.label:unknown}}
+        application.type: ${spring.cloud.dataflow.stream.app.type:unknown}
+        instance.index: ${vcap.application.instance_index:${spring.cloud.stream.instanceIndex:0}}
+        application.guid: ${spring.cloud.application.guid:unknown}
+
```

Comment:
Isn't it still appropriate for CF deploys to have the following variable defined?

```
      organization.name: ${vcap.application.org_name:default}
      space.id: ${vcap.application.space_id:unknown}
      space.name: ${vcap.application.space_name:unknown}
      application.id: ${vcap.application.application_id:unknown}
      application.version: ${vcap.application.application_version:unknown}```

Reply:
and wouldn't these be needed for batch apps on CF as well?

Reply:
The `common:`  is applied to all platforms, including CF but also local K8s. The `application.name` and  `instance.index` metrics tags are common (and required) for all platforms, while the `organization.name`, `space.id`, `space.name`, `application.id`, `application.version` tags are only CF specific and therefore defined under the `cloudfoundry:` platform prefix.